### PR TITLE
[Locales] Support ubuntu redhat

### PR DESCRIPTION
**Pull request by nervo**

### CODE REVIEW

File: manala.locales/lookup_plugins/manala_locales_codes.py [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+                })
+            else:
+
+                # Must be a dict
+                if not isinstance(code, dict):
+                    raise AnsibleError('Expect a dict')
+
+                # Check index key
+                if not code.has_key('code'):
+                    raise AnsibleError('Expect "code" key')
```
Comment (chalasr) [inline manala.locales/lookup_plugins/manala_locales_codes.py]: I suggest
> Expected "code" key to be set

or

> Missing "code" key

File: manala.locales/lookup_plugins/manala_locales_codes.py [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+            if isinstance(code, basestring):
+                item = itemDefault.copy()
+                item.update({
+                    'code': code
+                })
+            else:
+
+                # Must be a dict
+                if not isinstance(code, dict):
+                    raise AnsibleError('Expect a dict')
```
Comment (chalasr) [inline manala.locales/lookup_plugins/manala_locales_codes.py]: s/Expect/Expected everywhere (assertion occurred already)

Comment (nervo) [reply manala.locales/lookup_plugins/manala_locales_codes.py]: See: https://github.com/manala/ansible-roles/issues/97 (and thx for the review)

**Comment by chalasr** [comment]
Thanks for contributing @juju4!
Thanks for finishing @nervo

**Comment by juju4** [comment]
Thanks @nervo, @chalasr! it seems a bit more complicate ;-)

**Comment by nervo** [comment]
@juju4 I would say more elaborate, as it handles states (present/absent), check mode, and tests :)

**Comment by juju4** [comment]
yeah. was just kidding :)

**Comment by nervo** [comment]
Of course :)

**Pull request merged**